src/interface.jl

```diff
@@ -105,7 +105,7 @@ logabsdetjacinv(b::Bijector, y) = logabsdetjac(inv(b), y)
 ##############################
 
 struct Identity{N} <: Bijector{N} end
-(::Identity)(x) = x
+(::Identity)(x) = copy(x)
```

Comment:
Should this be copy or not? @mohamed82008 I noticed you had implemented some using `copy`, and I tried changing it to see if helped with the Zygote error (though it didn't seem to).

Reply:
Hmm not sure, if it worked previously then the `copy` is not necessary I suppose.